db/migrate/20160512191826_create_messages.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMessages < ActiveRecord::Migration
+  def change
+    create_table :messages do |t|
+      t.text :text
+      t.references :user, foreign_key: true
```

Comment:
This can work fine (I guess) but why do you ignore the way this is done elsewhere in the project?